src/libraries/Common/src/System/Net/Http/NoWriteNoSeekStreamContent.cs

```diff
@@ -60,10 +61,10 @@ Task SerializeToStreamAsync(Stream stream, TransportContext context, Cancellatio
             return copyTask;
         }
 
-#if HTTP_DLL
-        protected internal
-#else
+#if WIN_HTTP_HANDLER
```

Comment:
I would prefer this be called "WINHTTPHANDLER" or "WINHTTP_HANDLER".  Leaving it with 3 separate words is confusing since it won't make it obvious that this is about the System.Net.Http.WinHttpHandler.dll binary and related NuGet package.

Reply:
I personally find it more readable with every word separated but don't have a strong preference. Would WINHTTPHANDLER_DLL work to emphasize that this is a different assembly and not just a random switch?

Reply:
>Would WINHTTPHANDLER_DLL work to emphasize that this is a different assembly and not just a random switch?

That would work.

Reply:
Are you still planning on changing that define to 'WINHTTPHANDLER_DLL' ?

Reply:
Unless there are other suggestions, yes